pl-backend/Program.cs

```diff
@@ -6,6 +10,20 @@
 // Learn more about configuring Swagger/OpenAPI at https://aka.ms/aspnetcore/swashbuckle
 builder.Services.AddEndpointsApiExplorer();
 builder.Services.AddSwaggerGen();
+builder.Services.AddScoped<ITokenService, TokenService>();
+builder.Services.AddScoped<IUserService, UserService>();
+
```

Comment:
We also need this thing we inject in TokenService, i'll explain it later or you can check it yourself
```suggestion
builder.Services.AddSingleton<IHttpContextAccessor, HttpContextAccessor>();

```